scripts/build_multisig_tx.py

```diff
@@ -0,0 +1,212 @@
+"""
+Usage:
+    brownie run --network mainnet-fork build_multisig_tx main input.csv
+"""
+import csv
+import os
+from hashlib import sha256
+from typing import NamedTuple, TypedDict
+
+from ape_safe import ApeSafe, Safe
+from brownie import Contract, VestingEscrow, VestingEscrowFactory, VotingAdapter  # type: ignore
+from brownie.network.transaction import TransactionReceipt
+from eth_utils.conversions import to_bytes, to_hex
+from gnosis.safe import SafeTx
+from utils import log
+from web3 import Web3
+
+
+def main(csv_filename: str):
+    """Build vesting contracts deployment tx by parameters defined in CSV file"""
+    config = read_envs()
+    safe = ApeSafe(config["SAFE_ADDRESS"])
+    factory = VestingEscrowFactory.at(
+        address=config["FACTORY_ADDRESS"],
+        owner=safe.address,
+    )
+
+    # Uncomment for testing purpose
+    # factory = VestingEscrowFactory.at(
+    #     _test_factory_address(safe),
+    #     owner=safe.address,
+    # )
```

Comment:
Use flag instead of commented code for testing mode

Reply:
Brownie's scripts runner doesn't support providing flags, added as argument instead.